File: eSalon/eSalon.Model/Favorit.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eSalon.Model
{
    public class Favorit
    {
        public int FavoritId { get; set; }
        public int KorisnikId { get; set; }
        public int UslugaId { get; set; }
        public DateTime DatumDodavanja { get; set; }
        public string? UslugaNaziv { get; set; } 
        public decimal? Cijena { get; set; }
        public byte[]? Slika { get; set; }
        public int? Trajanje { get; set; }

    }
}

File: eSalon/UI/esalon_mobile/lib/models/favorit.dart
import 'package:json_annotation/json_annotation.dart';

part 'favorit.g.dart';

@JsonSerializable()
class Favorit {
  int? favoritId;
  int? korisnikId;
  int? uslugaId;
  DateTime? datumDodavanja;
  String? uslugaNaziv;
  double? cijena;
  String? slika; 
  int? trajanje;

  Favorit(
    this.favoritId,
    this.korisnikId,
    this.uslugaId,
    this.datumDodavanja,
    this.uslugaNaziv,
    this.cijena,
    this.slika,
    this.trajanje,
  );

  factory Favorit.fromJson(Map<String, dynamic> json) => _$FavoritFromJson(json);

  Map<String, dynamic> toJson() => _$FavoritToJson(this);
}
